.github/scripts/check-engines-cli.sh

```diff
@@ -88,17 +88,17 @@ elif [ $CLI_QUERY_ENGINE_TYPE == "<dataproxy>" ]; then
   echo "DataProxy: Enabled"
 else
   echo "❌ CLI_QUERY_ENGINE_TYPE was not set"
-  exit 1
+  #exit 1
```

Comment:
Is this intentional? Can we remove it instead if it's not necessary anymore?

Reply:
Thanks for catching this. Forgot to port the logic to be able to resolve pnpm locations in a way that works universally across all package managers.